Okta.Xamarin/Okta.Xamarin/OktaStateManager.cs

```diff
@@ -80,6 +87,30 @@ public bool IsAuthenticated
             }
         }
 
+        /// <summary>
+        /// Gets or sets the last response received from the API.
+        /// </summary>
+        public HttpResponseMessage LastApiResponse { get; set; }
+
+        /// <summary>
+        /// Gets the token of the specified type.
+        /// </summary>
+        /// <param name="tokenType">The token type.</param>
+        /// <returns>string</returns>
+        public string GetToken(TokenType tokenType)
+        {
+            switch (tokenType)
```

Comment:
Should we also include Id token?

Reply:
Yes, I noticed that it was missing in later changes.  It will be in a later PR.